src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/SeaCreatureTracker.kt

```diff
@@ -27,6 +32,7 @@ object SeaCreatureTracker {
 
     private val tracker = SkyHanniTracker("Sea Creature Tracker", { Data() }, { it.fishing.seaCreatureTracker })
     { drawDisplay(it) }
+    private val armorNames = mutableListOf<String>()
```

Comment:
Prefer immutable data structures wherever possible. You can also use a `Set` here instead, for faster lookups.

```suggestion
    private var armorNames = setOf<NEUInternalName>()
```